rootfs/ingest/bin/replctl

```diff
@@ -31,21 +31,22 @@ Index table columns using replication service to Qserv
 #  Imports of standard modules --
 # -------------------------------
 import argparse
-from enum import Enum, auto
 import logging
 import os
 import sys
+from enum import Enum
 
 # ----------------------------
 # Imports for other modules --
 # ----------------------------
+import qserv.util as util
+from qserv.contribqueue import QueueManager
```

Comment:
The right order of the import statement should be:
```
from qserv.contribqueue import QueueManager
from qserv.ingest import Ingester
from qserv.jsonparser import DatabaseStatus
from qserv.metadata import ContributionMetadata
import qserv.util as util
```

Reply:
:+1: